core/bones/spatial.py

```diff
@@ -54,10 +54,18 @@ def __init__(self, *, boundsLat: Tuple[int, int], boundsLng: Tuple[int, int], gr
             :param gridDimensions: Number of sub-regions the map will be divided in
         """
         super().__init__(**kwargs)
-        assert isinstance(boundsLat, tuple) and len(boundsLat) == 2, "boundsLat must be a tuple of (int, int)"
-        assert isinstance(boundsLng, tuple) and len(boundsLng) == 2, "boundsLng must be a tuple of (int, int)"
+        assert isinstance(boundsLat, tuple) and len(boundsLat) == 2, "boundsLat must be a tuple of (float, float)"
+        assert isinstance(boundsLng, tuple) and len(boundsLng) == 2, "boundsLng must be a tuple of (float, float)"
         assert isinstance(gridDimensions, tuple) and len(
             gridDimensions) == 2, "gridDimensions must be a tuple of (int, int)"
+        """
+        Checks if boundsLat and boundsLng have possible values
+        See https://docs.mapbox.com/help/glossary/lat-lon/
+        """
```

Comment:
Can we turn this into a standard comment?
```suggestion
        # Checks if boundsLat and boundsLng have possible values
        # See https://docs.mapbox.com/help/glossary/lat-lon/
```